src/org/infinity/gui/StructViewer.java

```diff
@@ -408,10 +409,16 @@ public Component getTableCellRendererComponent(JTable table, Object value,
   @Override
   public void actionPerformed(ActionEvent event)
   {
+    final String cmd = event.getActionCommand();
```

Comment:
Theoretically the result of this method can be `null` therefore I swap operands in the `if`s. If the static code analyzer sometime will be used, it will not trigger on these places